assets/js/blocks/legacy-template/test/utils.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Internal dependencies
+ */
+import { beginsWith, getMatchingTemplateData } from '../utils';
+
+export const TEMPLATES = {
```

Comment:
Isn't this duplicated? Why don't we import it from `constants`? Is it because of the localization?

Reply:
Yeah its duplicated, I wanted to isolate this as test data only. I did consider importing the actual data but if the production data changes for whatever reason it might cause a false negative if its causes the unit tests to fail.

Reply:
🤔  Mm, ok. But wouldn't we want it to fail if the production data changes? Like, wouldn't that mean that our matching logic doesn't work and we need to take another look at it?

Reply:
>But wouldn't we want it to fail if the production data changes?

I'm in two minds about this, because I get understand your point but in that scenario we are testing the data not the function. Let me try and update and see how this goes.

Reply:
Updated 👌🏻

Reply:
Now I see what you mean and I understand your conundrum. Thanks for elaborating.